diagnostic.md

```diff
@@ -6,70 +6,87 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What are two ways of telling rails where to route a GET request to '/books/4'?
 
 ```rb
-// your response here
+1.
+#!/bin/bash
+API="${API_ORIGIN:-http://localhost:4741}"
+URL_PATH="/books/4"
+curl "${API}${URL_PATH}" \
+  --include \
+  --request GET
+echo
+
+2.
+Using curl in the terminal
 ```
 
 In what file does the above code go?
 
 ```md
-// your response here
+1. scripts/books/show.sh
+2. Terminal
 ```
 
 In your own words, what does the command `bundle exec rake db:migrate` do?
 
 ```md
-// your response here
+Runs migrations for the current environment that have not run yet.
+By default it will run migrations only in the development environment.
+It will automaticlly create a Schema.
 ```
 
 What is the difference between `rails g scaffold things` and
 `rails g controller things`?
 
 ```md
-// your response here
+rails g controller things will only generate a controller for a table.
+scaffold will generate a set of MVC (model, view, controller) for a table.
```

Comment:
`rails g scaffold things` will generate a migration, model, routes, serializer, controller, and tests for `things`.
`rails g controller things` will generate a controller and tests for `things`.